### PR TITLE
Added field-descriptor metadata documentation

### PR DESCRIPTION
**Pull request by wachterjohannes**
Main PR: https://github.com/sulu-io/sulu/pull/1991 and https://github.com/sulu-io/sulu/pull/2000

### CODE REVIEW

File: bundles/core/list-builder/filter-metadata.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
+Filter Metadata
+===============
+
+The filter metadata extends the basic metadata to add information about filter
+form. Each property can have a input-type and parameters which will be use to
```
Comment (danrot) [inline bundles/core/list-builder/filter-metadata.rst]: `an input-type`

Comment (danrot) [inline bundles/core/list-builder/filter-metadata.rst]: `parameters, which will be used to`

File: bundles/core/list-builder/filter-metadata.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        <filter-type-parameters>
+            <filter:parameter key="prefetchUrl">
+                <![CDATA[http://sulu.lo/admin/api/contact/titles]]>
+            </filter:parameter>
+            <filter:parameter key="resultKey">contactTitles</filter:parameter>
+            <filter:parameter key="valueKey">title</filter:parameter>
+        </filter-type-parameters>
+    </identity-property>
+
+This Example creates a input where the user can select a title from the given
```
Comment (danrot) [inline bundles/core/list-builder/filter-metadata.rst]: `This example`

**Comment by wachterjohannes** [comment]
@danrot we talked about leaving that PR unmerged

**Pull request closed without merging**